scout/utils/acmg.py

```diff
@@ -212,6 +212,8 @@ def get_acmg(acmg_terms):
                     if term.startswith(prefix):
                         term_list.append(term)
                         break
+                else:
```

Comment:
I will say I found the original code, without AI style suffix and prefix maps easier to follow. This is elegant, but elegant pays a price in need for focus to modify. Dirt is very visible on a shiny surface I guess. 😆